app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ProjectsController < ApplicationController
+  def show
+    @project = Project.find(params[:id])
+    @challenge = Challenge.find_by_project(@project.id)
+  end
+end
```

Comment:
A convention in rails is that we want to send over as few objects to the view as possible, so since you can access the challenge you want off of the `project` object, you only need to send that `project` object to the view.